flujo/cli/main.py

```
@@ -1945,6 +2000,103 @@ def _resolve(rule_id: str) -> Optional[str]:
+                from ..validation.fixers import plan_fixes, apply_fixes_to_file, build_fix_patch
+
+                # Parse per-rule filter from env (comma-separated globs), e.g., "V-T1,V-C2*"
+                rule_filter = None
+                try:
+                    if fix_rules:
+                        rule_filter = [x.strip() for x in fix_rules.split(",") if x.strip()]
+                    else:
+                        rf = _os.getenv("FLUJO_FIX_RULES")
+                        if rf:
+                            rule_filter = [x.strip() for x in rf.split(",") if x.strip()]
+                except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)